src/auditor/auditor.py

```diff
@@ -243,13 +244,21 @@ def read_metatable(self, table, fields):
                     table_sgid_name, table_agol_itemid, table_agol_name, table_category = row
                     table_authoritative = 'n'
 
-                if table_agol_itemid:  #: Don't evaluate null itemids
-                    if table_agol_itemid not in self.metatable_dict:
-                        self.metatable_dict[table_agol_itemid] = [
-                            table_sgid_name, table_agol_name, table_category, table_authoritative
-                        ]
-                    else:
-                        duplicate_keys.append(table_agol_itemid)
+                #: Item IDs are UUIDs. If we can't parse the item id listed in the table, it means the layer is not
+                #: in AGOL and this row should be skipped (catches both magic words and empty entries)
+                try:
+                    uuid.UUID(table_agol_itemid)
+                except (AttributeError, ValueError, TypeError):
+                    if self.verbose:
+                        print(f'{table_sgid_name} from {table} table not in AGOL')
```

Comment:
**suggestion**: add the `table_agol_itemid` to the log message

Reply:
To be honest, I only put the print in here for manual testing (and yes, I know a proper automated test would be better here, but everything's still too tightly coupled to make that a simple task). I'm more inclined to just remove it completely than to clutter the log with a weekly report of what items we're skipping in the metatable. Thoughts?

Reply:
good points. I agree with you.